db/migrate/20170329064703_create_user_emails.rb

```diff
@@ -0,0 +1,31 @@
+class CreateUserEmails < ActiveRecord::Migration
```

Comment:
Done - I grouped them I think a little more logically than just "one migration modifies one table" let me know what you think